persistqueue/tests/test_sqlqueue.py

```diff
@@ -234,14 +234,33 @@ def test_json_serializer(self):
                 e=[1]
             ))
         q.put(x)
-        self.assertEquals(q.get(), x)
+        self.assertEqual(q.get(), x)
 
     def test_put_0(self):
         q = SQLiteQueue(path=self.path)
         q.put(0)
         d = q.get(block=False)
         self.assertIsNotNone(d)
 
+    def test_get_id(self):
+        q = SQLiteQueue(path=self.path)
+        q.put("val1")
+        val2 = q.put("val2")
+        q.put("val3")
+        item = q.get(item=val2)
```

Comment:
use`(id=val2)`

Reply:
Ah, dang.. I had renamed those to make it simpler. Guess that's the point of unit tests.  Doesn't work if you don't rerun them. lol